Buildings/Applications/DHC/EnergyTransferStations/CoolingIndirect.mo

```diff
@@ -0,0 +1,287 @@
+  connect(port_b2, TBuiRet.port_b)
+    annotation (Line(points={{-100,-60},{-90,-60}}, color={0,127,255}));
+  connect(senTDisRet.port_b, port_b1)
+    annotation (Line(points={{90,60},{100,60}}, color={0,127,255}));
+  connect(powCal.y, Q_flow)
+    annotation (Line(points={{-16,-90},{90,-90},{90,-130}}, color={0,0,127}));
+  connect(powCal.y, int.u) annotation (Line(points={{-16,-90},{-10,-90},{-10,-110},
+          {-2,-110}}, color={0,0,127}));
+  connect(int.y, Q)
+    annotation (Line(points={{21,-110},{50,-110},{50,-130}}, color={0,0,127}));
+  connect(TSet, con.u_s)
+    annotation (Line(points={{-120,0},{-92,0}}, color={0,0,127}));
+  connect(con.u_m, TBuiRet.T)
+    annotation (Line(points={{-80,-12},{-80,-49}}, color={0,0,127}));
+  connect(con.y, val.y)
+    annotation (Line(points={{-69,0},{-20,0},{-20,48}}, color={0,0,127}));
+  connect(TBuiRet.port_a, hex.port_b2) annotation (Line(points={{-70,-60},{0,-60},
+          {0,-6},{20,-6}}, color={0,127,255}));
+  annotation (Icon(coordinateSystem(preserveAspectRatio=false), graphics={
+        Rectangle(
+          extent={{-100,-56},{100,-64}},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Rectangle(
+          extent={{-100,64},{100,56}},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Rectangle(
+          extent={{-80,80},{80,-80}},
+          lineColor={175,175,175},
+          fillColor={35,138,255},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-52,40},{54,-40}},
+          lineColor={0,0,0},
+          fillColor={35,138,255},
+          fillPattern=FillPattern.Solid,
+          textStyle={TextStyle.Bold},
+          textString="ETS")}),                                   Diagram(
+        coordinateSystem(preserveAspectRatio=false, extent={{-100,-120},{100,100}})),
+    Documentation(info="<html>
+<p>
+Indirect cooling energy transfer station (ETS) model that controls the building chilled water supply temperature 
+by modulating a primary control valve on the district supply side. The design is based on a typical district 
+cooling ETS described in ASHRAE's 
+<a href=\"https://www.ashrae.org/technical-resources/bookstore/district-heating-and-cooling-guides\">
+District Cooling Guide</a>.  
+As shown in the figure below, the building pumping design (constant, variable) is specified on the building
+side, not within the ETS. 
+</p>
+<p align=\"center\">
+<img src=\"modelica://Buildings/Resources/Images/Applications/DHC/EnergyTransferStations/CoolingIndirect.PNG\"/>
+</p>
+<h4>Reference</h4>
+<p>
+American Society of Heating, Refrigeration and Air-Conditioning Engineers. (2013). Chapter 5: End User Interface. In <i>District Cooling Guide</i>. 1st Edition. 
+</p>
+</html>", revisions="<html>
+<ul>
+<li>November 1, 2019, by Kathryn Hinkelman:<br>First implementation. </li>
```

Comment:
change line:
```
<li>
November 1, 2019, by Kathryn Hinkelman:<br>
First implementation. 
</li>
```

Reply:
Done. Thanks for the tip on the unit tests.